.github/workflows/build.yaml

```diff
@@ -0,0 +1,118 @@
+name: build
+
+env:
+  # The name of the main module repository
+  main_project_module: app
+
+
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+#  release:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Could you please put a comment on what checkout@v3 refers to?

Reply:
No idea but it works 
![image](https://github.com/EPFL-SwEnt-2024-LaStartUp/TripTracker/assets/79469048/b08e2b40-a3a8-4f95-ae09-75670f3f6622)

Reply:
Okay, fair enough